### ISSUE TITLE
F55 link name is truncated to "F55: received"

### ISSUE THREAD

**Issue by filippo-zorzi** [w3c/wcag#3866]
All the links to the Failure F55 are labelled as "F55: received" instead of "F55: Failure of Success Criteria 2.1.1, 2.4.7, 2.4.13, and 3.2.1 due to using script to remove focus when focus is received".

<img width="921" alt="Screenshot 2024-05-17 at 09 55 42" src="https://github.com/w3c/wcag/assets/165811341/e040da79-c2f8-450a-9bcb-01cd891ebe05">

This problem is present in:
 - 2.1.1
 - 2.4.7
 - 2.4.13
 - 3.2.1
